### PR TITLE
Removed legacy CM Camera info

### PR DESCRIPTION
**Pull request by nathan-contino**
* Follow-up to https://github.com/raspberrypi/documentation/pull/3188
* Should close https://github.com/raspberrypi/documentation/issues/3192

### CODE REVIEW

**Comment by aallan** [comment]
@6by9 can you take a peek?

File: documentation/asciidoc/computers/configuration/pin-configuration.adoc
Comment (aallan) [inline documentation/asciidoc/computers/configuration/pin-configuration.adoc:1]: @pelwell Does this stuff still apply in Bookworm?

Comment (6by9) [reply documentation/asciidoc/computers/configuration/pin-configuration.adoc:1]: As per my comment on 3188

> Other than configuring the legacy camera and DSI display on CMs, dt-blobs should not be needed or used. The main downside it that they totally replace the one built into the firmware, therefore it breaks if there is ever a change of syntax required.

It only has a couple of very niche use cases, but is applicable.

Comment (pelwell) [reply documentation/asciidoc/computers/configuration/pin-configuration.adoc:1]: The dtparam changes look correct and are still needed in/orthogonal to Bookworm. The dtblob.bin changes are still as deprecated in Bookworm as they were in Bullseye.

**Comment by 6by9** [comment]
Other than the commit text not describing all the changes, it looks reasonable.

**Review by aallan — APPROVED** [review]

File: documentation/asciidoc/computers/configuration/pin-configuration.adoc
Comment (aallan) [reply documentation/asciidoc/computers/configuration/pin-configuration.adoc:1]: @nathan-contino Okay, we should edit the bit where it's for "advanced users" and remove the warning around the firmware being newer than 2014 (as 2014 was now a long time ago) and throw in a `WARNING` block about deprecation at the top. Then we'll ship this, and it'll be correct'er than it was.

Comment (nathan-contino) [reply documentation/asciidoc/computers/configuration/pin-configuration.adoc:1]: Done and done.

File: documentation/asciidoc/computers/configuration/pin-configuration.adoc [hunk 1]
```diff
@@ -24,7 +22,7 @@ Note that it may take a few seconds to get from stage 1 to stage 4. During that
 * Delay the peripheral's startup until stage 4/5 has been reached
 * Add an appropriate pull-up/-down resistor
 
-=== Providing a Custom Device Tree Blob
+=== Provide a custom Device Tree blob
```
Comment (lurch) [inline documentation/asciidoc/computers/configuration/pin-configuration.adoc:25]: There's a bit a few lines above here where it says `dtblob.bin` when it should be `dt-blob.bin`.

**Comment by lurch** [comment]
One minor comment, but otherwise looks fine to me :+1:

**Comment by aallan** [comment]
Okay we're ready to go?

**Comment by nathan-contino** [comment]
Indeed

File: documentation/asciidoc/computers/configuration/pin-configuration.adoc [hunk 2]
```diff
@@ -35,12 +33,12 @@ sudo dtc -I dts -O dtb -o /boot/firmware/dt-blob.bin dt-blob.dts
 Similarly, a `.dtb` file can be converted back to a `.dts` file, if required.
 
 ----
-dtc -I dtb -O dts -o dt-blob.dts /boot/firmware/dt-blob.bin
+dtc -I dtb -O dts -o dt-.dts /boot/firmware/dt-.bin
```
Comment (lurch) [inline documentation/asciidoc/computers/configuration/pin-configuration.adoc:36]: Eeek, what happened here @nathan-contino ??

Comment (nathan-contino) [reply documentation/asciidoc/computers/configuration/pin-configuration.adoc:36]: Egad. I'm going to blame the GitHub editor for this nonsense. My bad for claiming "Indeed" to Al's question before.

Comment (nathan-contino) [reply documentation/asciidoc/computers/configuration/pin-configuration.adoc:36]: Opened https://github.com/raspberrypi/documentation/pull/3194 to fix.

**Pull request merged**